core/src/test/java/com/graphhopper/routing/util/parsers/RacingBikeTagParserTest.java

```diff
@@ -73,45 +71,45 @@ public void testAvoidTunnel() {
         ReaderWay osmWay = new ReaderWay(1);
         osmWay.setTag("highway", "residential");
         osmWay.setTag("tunnel", "yes");
-        assertPriorityAndSpeed(SLIGHT_AVOID, 18, osmWay);
+        assertPriorityAndSpeed(0.7, 18, osmWay);
 
         osmWay.setTag("highway", "secondary");
         osmWay.setTag("tunnel", "yes");
-        assertPriorityAndSpeed(UNCHANGED, 24, osmWay);
+        assertPriorityAndSpeed(1.0, 24, osmWay);
 
         osmWay.setTag("bicycle", "designated");
-        assertPriorityAndSpeed(PREFER, 24, osmWay);
+        assertPriorityAndSpeed(1.2, 24, osmWay);
     }
 
     @Test
     @Override
     public void testService() {
         ReaderWay way = new ReaderWay(1);
         way.setTag("highway", "service");
-        assertPriorityAndSpeed(SLIGHT_AVOID, 12, way);
+        assertPriorityAndSpeed(0.7, 12, way);
```

Comment:
Note: Was SLIGHT_AVOID (0.9) before. 0.7 is probably also OK, but I would prefer 0.8.